api/controllers/v1/github.ts

```diff
@@ -77,10 +84,10 @@ export const TriggerBuild = async (event: any = {}, context: any = {}): Promise<
   }
 
   const env = c.get<string>('env');
-  const repoInfo = await repoBranchesRepository.getRepo(body.repository.name);
+  const repoInfo = await docsetsRepository.getRepo(body.repository.name);
   const jobPrefix = repoInfo?.prefix ? repoInfo['prefix'][env] : '';
   // TODO: Make job be of type Job
-  const job = await prepGithubPushPayload(body, repoBranchesRepository, jobPrefix);
+  const job = await prepGithubPushPayload(body, repoBranchesRepository, docsetsRepository, jobPrefix);
```

Comment:
we are querying for `getRepo` twice (here and in `getRepoBranchesByRepoName`) can pass the repoInfo as a param

Reply:
Nice, done!